### PR TITLE
test custom images

### PR DESCRIPTION
**Pull request by razvan**
- **feat(test): custom images**
- **update test-definition.yaml**

# Description

*Please add a description here. This will become the commit message of the merge request later.*

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] CRD documentation for all fields, following the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs-style-guide).
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
- [ ] Changes need to be "offline" compatible
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] Code contains useful logging statements
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated. Follows the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs-style-guide).
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
- [ ] [Roadmap](https://github.com/orgs/stackabletech/projects/25/views/1) has been updated
```

### CODE REVIEW

File: tests/templates/kuttl/orphaned-resources/30-install-airflow-cluster.yaml.j2 [hunk 1]
```diff
@@ -26,8 +26,12 @@ metadata:
   name: airflow
 spec:
   image:
+{% if test_scenario['values']['airflow-latest'].find(",") > 0 %}
+    custom: "{{ test_scenario['values']['airflow-latest'].split(',')[1] }}"
+    productVersion: "{{ test_scenario['values']['airflow-latest'].split(',')[0] }}"
+{% else %}
     productVersion: "{{ test_scenario['values']['airflow-latest'] }}"
-    pullPolicy: IfNotPresent
```
Comment (dervoeti) [inline tests/templates/kuttl/orphaned-resources/30-install-airflow-cluster.yaml.j2:30]: Was the removal of this line intended?

**Review by dervoeti — COMMENTED** [review]
One small thing, rest looks fine!

File: tests/templates/kuttl/orphaned-resources/30-install-airflow-cluster.yaml.j2 (hunk 1 above)
Comment (razvan) [reply tests/templates/kuttl/orphaned-resources/30-install-airflow-cluster.yaml.j2:30]: reverted https://github.com/stackabletech/airflow-operator/pull/465/commits/c0e11419aaebc1c98a604cc56c8a9db6a68ecb55

**Review by dervoeti — APPROVED** [review]
LGTM

**Pull request merged**